osv2/search.go

```diff
@@ -115,20 +150,41 @@ func (r *SearchRequest) WithQuery(q opensearchtools.Query) *SearchRequest {
 // fromDomainSearchRequest creates a new SearchRequest from the given [opensearchtools.SearchRequest]
 func fromDomainSearchRequest(req *opensearchtools.SearchRequest) (SearchRequest, opensearchtools.ValidationResults) {
 	vrs := opensearchtools.NewValidationResults()
-	var searchRequest SearchRequest
+	var (
+		searchRequest SearchRequest
+		v2Aggs        map[string]opensearchtools.Aggregation
```

Comment:
This code is already in the `osv2` sub-package so I feel like the `v2` prefixes on these variables are unnecessary.